frontend/src/main.js

```diff
@@ -3,6 +3,7 @@ import App from "./App.vue";
 import router from "./router";
 
 import "./assets/main.css";
+import "./assets/tailwind.css";
```

Comment:
Since I can't comment on tailwind.css (file too large), my comments are here:

[Tailwind](https://tailwindcss.com/docs/installation) is a npm package to make css easier. This css would have been auto generated from a tailwind config and other css. I'm fine with using tailwind, but you'll have to get it set up correctly so that we can re-build this file as necessary. 65,000 lines of css might be slightly painful to maintain.

The other (easier) option would just be to pull out the styles you need from this file and put it in the components themselves, and delete this file.